Dw/orion (#26)
davwheat/personal-portfolio: blog/2022/01/22/o2-orion-monopoles.mdx

---
title: 'O2 Beacon 3 - Orion'
description:

>   O2's latest flagship monopoles, Orion, are a mastermind in the mobile networking world, cramming two 5G carriers and tonnes of 4G bands into a

Reviewer: ```suggestion
  O2's latest flagship monopoles, Orion, are a mastermind in the mobile networking world, cramming two 5G carriers and tonnes of 4G bands into a single streetworks cabinet and monopole. It's a work of art.
```

>   single streetworks cabinet and monopole. It's a work of art.

Reviewer: ```suggestion
```
Remove this line for formatting as there was an unnecessary line break.

path: 2022/01/07/o2-beacon-3-orion
created_at: 2022-01-22T23:00Z
---

import { FactBox, MathBlock, Tooltip, SpectrumMap } from '@blog'

<TableOfContents {...props} />

<FactBox title="Note">

> Click on any of the images in this article to view them at their full-size glory.

Reviewer: ```suggestion
Click on any of the images in this article to view them in their full-size glory.
```

</FactBox>

# A brief history

> O2 (<Tooltip text="MCC: 234, MNC: 10"><code>234/10</code></Tooltip>) is the largest network operator in the United Kingdom in terms of customers.

Reviewer: ```suggestion
O2 (<Tooltip text="MCC: 234, MNC: 10"><code>234/10</code></Tooltip>) is the largest mobile network operator in the United Kingdom in terms of customers.
```

> Including its <Tooltip text="Mobile virtual network operators">MVNOs</Tooltip>, O2's customer base totals a whopping 36.2 million customers in a

Reviewer: ```suggestion
Including its <Tooltip text="Mobile virtual network operators">MVNOs</Tooltip>, O2's customer base totals a whopping 36.2 million customers in a country of just 67 million, meaning they have a market share of 54%.
```

> country of just 67 million, meaning they have a market share of 54%.

Reviewer: ```suggestion
```

([Source](https://news.o2.co.uk/press-release/o2-continues-customer-growth-and-profitability-as-it-supports-uk-through-pandemic))

> In the past, O2 has had a fairly troublesome history in terms of capacity. Many parts of the UK which used to be more rural have expanded over

Reviewer: ```suggestion
In the past, O2 has struggled in terms of capacity. Many parts of the UK which used to be more rural have grown over time, resulting in O2 sites which used to be more than capable of providing decent performance to customers within their zone suffering from lower performance as network load increases each year. This is extremely prevalent with their main 800 MHz LTE deployments across rural and suburban communities.
```
UK is obvious and doesn't need restated, tried to make the rest a little more positive.

> time, resulting in O2 sites which used to be more than capable of providing decent performance to customers within their zone suffering from

Reviewer: ```suggestion
```

> lower performance as network load increases each year. This is extremely prevalent with their main deployments being 800 MHz LTE across rural and

Reviewer: ```suggestion
```

> suburban United Kingdom.

Reviewer: ```suggestion
```

# Introduction to Orion

> In 2021, O2 launched its **Beacon 3** network upgrade project, including its brand new, flagship Orion monopoles.

Reviewer: ```suggestion
In 2021, O2 launched its "Beacon 3" network upgrade project, including its brand new, flagship Orion monopoles.
```

Quotes not bold for titles.

<FactBox>

"Beacon" was the codename for a series of joint upgrade plans made between O2 and Vodafone to provide LTE to the whole of the UK.

> Beacon 3 is the first Beacon project to include installation of <Tooltip text="one network operator">unilateral</Tooltip> sites as opposed to the

Reviewer: ```suggestion
Beacon 3 is the first Beacon project to include installation of <Tooltip text="one network operator">unilateral</Tooltip> sites as opposed to the multi-operator radio access network (MORAN) sites we typically see with Cornerstone Networks.
```

> multi-operator radio access network (MORAN) sites we typically see with Cornerstone Networks.

Reviewer: ```suggestion
```

> This indicates a possibly deteriorating relationship between the two operators.

Reviewer: ```suggestion
There has been speculation about the future relationship between O2 and Vodafone as a result of these changes. For example, their needs may be diverging, or there may be other business factors.
```

Evidence for original claim?

</FactBox>

> Orion monopoles manage to cram a multitude of 4G bands, along with n28 and n78 5G and their standard 2G/3G all into **one street cabinet and one

Reviewer: ```suggestion
Orion monopoles manage to cram a multitude of 4G bands, along with n28 and n78 5G and their standard 2G/3G all into **one street cabinet and one monopole**, often replacing what used to be basic 4G/3G/2G monopoles and several street cabinets.
```

> monopole**, often replacing what used to be basic 4G/3G/2G monopoles and several street cabinets.

Reviewer: ```suggestion
```

> O2 have begun favouring these new deployments because of their simplicity. A **pole swap can be done in just one day**, consisting of contractors

Reviewer: ```suggestion
O2 have begun favouring these new deployments because of their simplicity. A pole swap can be done in just one day, consisting of contractors installing the feeders in the new cabinet, and an HIAB truck arriving with the new monopole, lifting and dropping it into place, and then taking the old one away to be stripped for parts.
```

Bold doesn't make much sense here.

> installing the feeders in the new cabinet, and an HIAB truck arriving with the new monopole, lifting and dropping it into place, and then taking

Reviewer: ```suggestion
```

> the old one away to be stripped for parts.

Reviewer: ```suggestion
```

![An O2 Orion monopole alongside a previous Beacon 2 monopole](./orion_vs_old.jpg)

# Planning applications

> For mobile networks to install new street infrastructure, they do not need to apply for planning permission. Instead, they must submit a notice

Reviewer: ```suggestion
For mobile networks to install new street infrastructure, they do not need to apply for planning permission. Instead, they must submit a notice of prior notification. This is a notice of intent, basically saying "I want to do something. Tell me if I can't."
```

> of **prior notification**. This is a notice of intent, basically saying "I want to do something. Tell me if I can't.".

Reviewer: ```suggestion
```

> The local planning authority has **56 days to object to the plans, or approve them**. No response after 56 days is classed as an approval. The

Reviewer: ```suggestion
The local planning authority has 56 days to object to the plans, or approve them. No response after 56 days is classed as an approval. The big kick against conspiracy theorists is that **LPAs may only reject proposed development due to its siting or appearance**.
```

> big kick against conspiracy theorists is that **LPAs may only reject proposed development due to its siting or appearance**.

Reviewer: ```suggestion
```

> The compact nature of Orion monopole sites mean that councils are more likely to approve them compared to monopoles from other operators which

Reviewer: ```suggestion
The compact nature of Orion monopole sites mean that councils are more likely to approve them compared to monopoles from other operators which may clutter public areas with various street furniture.
```

Do you have any source? Everyone is using compact monopoles.

> may clutter public areas with various street furniture.

Reviewer: ```suggestion
```

## Exemptions

> Exemptions exist from submitting prior notification plans, aimed to make minor changes easier for network operators. While they are meant for

Reviewer: ```suggestion
Exemptions exist from submitting prior notification plans, aimed to make minor changes easier for network operators. While they are meant for swapping out equipment when it fails or for adding new microwave link dishes to sites, O2 have now been seen using these powers for performing full-on Orion site swaps in some areas.
```

> swapping out equipment when it fails or for adding new microwave link dishes to sites, O2 have now been seen using these powers for performing

Reviewer: ```suggestion
```

> full-on Orion site swaps in some areas.

Reviewer: ```suggestion
```

> One of my local monopole sites in Burgess Hill, for example, has recently been swapped from a Beacon 2 site to a new dual operator Orion with no

Reviewer: ```suggestion
One of my local monopole sites in Burgess Hill, for example, has recently been swapped from a Beacon 2 site to a new dual operator Orion with no prior notice submitted. This was possible due to one exemption:
```

> prior notice submitted. This was possible due to one exemption:

Reviewer: ```suggestion
```

> > ...prior approval does not apply to ground-based masts which when altered or replaced would not exceed the greater of the height of the

Reviewer: ```suggestion
> ...prior approval does not apply to ground-based masts which when altered or replaced would not exceed the greater of the height of the existing mast and 20m above ground level.
```

> > existing mast and 20m above ground level.

Reviewer: ```suggestion
```

> Since my old local monopole was 20m in height, and so was the new Orion monopole, **the site could be swapped out without any form of approval

Reviewer: ```suggestion
Since my old local monopole was 20m in height, and so was the new Orion monopole, the site could be swapped out **without any form of approval required**.
```

> required**.

Reviewer: ```suggestion
```

# Orion equipment

> Orion installations differ between vendor zones. The majority of sites we've seen have been O2 <Tooltip text="only O2">unilaterals</Tooltip>

Reviewer: ```suggestion
Orion installations differ between vendor zones. The majority of sites we've seen have been O2 <Tooltip text="only O2">unilaterals</Tooltip> using Ericsson equipment. We've seen some Cornerstone shared sites (where O2 and Vodafone are both located on the monopole), though.
```

> using Ericsson equipment. We've seen some Cornerstone shared sites (where O2 and Vodafone are both located on the monopole), though.

Reviewer: ```suggestion
```

## Nokia vendor

> [My site](https://www.google.com/maps/place/50%C2%B058'00.2%22N+0%C2%B008'13.7%22W/@50.9667335,-0.1376914,19z) was one of the first we'd seen in

Reviewer: ```suggestion
[My site](https://www.google.com/maps/place/50%C2%B058'00.2%22N+0%C2%B008'13.7%22W/@50.9667335,-0.1376914,19z) was one of the first we'd seen in the wild using Nokia equipment. It was also a dual-operator site, with O2 and Vodafone both broadcasting from it.
```

> the wild using Nokia equipment. It was also a dual-operator site, with O2 and Vodafone both broadcasting from it.

Reviewer: ```suggestion
```

> Unlike the [previous Ericsson sites seen by Jake and Peter in Cardiff](https://www.youtube.com/watch?v=ou_CGjbQ8uc), this site doesn't appear to

Reviewer: ```suggestion
Unlike the [previous Ericsson sites seen by Jake and Peter in Cardiff](https://www.youtube.com/watch?v=ou_CGjbQ8uc), this site doesn't appear to be equipped for a massive MIMO deployment, instead preferring 8T8R for n78 5G.
```

"kitted out" is slang that isn't globally used

> be kitted out for a massive MIMO deployment, instead preferring 8T8R for n78 5G.

Reviewer: ```suggestion
```

> On the top of the monopole are three

Reviewer: ```suggestion
On the top of the monopole are three [Commscope 24-port RRZZT4S4-65B-R6 antennas](https://www.commscope.com/product-type/antennas/base-station-antennas-equipment/base-station-antennas/itemrrzzt4s4-65b-r6/), each fully loaded with feeder cables. This is different from the Huawei passive antennas we had previously seen in Ericsson zones.
```

IDK why but the interjection there is still a bit grating to me. Thoughts?

> [Commscope 24-port RRZZT4S4-65B-R6 antennas](https://www.commscope.com/product-type/antennas/base-station-antennas-equipment/base-station-antennas/itemrrzzt4s4-65b-r6/)

Reviewer: ```suggestion
```

> (that's a mouthful), each fully loaded with feeder cables. This is different from the Huawei passive antennas we had previously seen in Ericsson

Reviewer: ```suggestion
```

> zones.

Reviewer: ```suggestion
```

> The street furniture used was a single Yorkshire cabinet, while the previous three cabinets (one for each operator, plus one for power) were

Reviewer: ```suggestion
The street furniture used was a single Yorkshire cabinet, while the previous three cabinets (one for each operator, plus one for power) were removed as part of the upgrade on day four.
```
Numbers ten and under should be spelled out, day isn't a proper noun in this context.

> removed as part of the upgrade on Day 4.

Reviewer: ```suggestion
```

![The new Commscope Yorkshire cabinet used in this Orion deployment](./commscope_cabinet.jpg)

<FactBox title="It's worth knowing...">

> The use of non-Huawei antennas in this Orion build **is not** related to the UK's ban on Huawei equipment in new 5G deployments, as that ban only

Reviewer: ```suggestion
The use of non-Huawei antennas in this Orion build **is not** related to the UK's ban on Huawei equipment in new 5G deployments, as that ban only applies to active equipment.
```

> applies to active equipment.

Reviewer: ```suggestion
```

</FactBox>

This particular Orion site appears to be configured for:

- 5G

>   - 8T8R 3500 MHz NR (n78) at 40 MHz

Reviewer: ```suggestion
  - 8T8R 3500 MHz NR (n78) at 40 MHz total
```

>   - 2T4R 700 MHz NR (n28) at 10 MHz

Reviewer: ```suggestion
  - 2T4R 700 MHz NR (n28) at 10 MHz, paired
```

Using consistent terms to what you use for LTE

- 4G
  - 4T4R 2100 MHz LTE (B1) at 10 MHz, paired
  - 2T2R 900 MHz LTE (B8) at 10 MHz, paired
  - 2T2R 800 MHz LTE (B20) at 10 MHz, paired
  - 8T8R 2300 MHz LTE (B40) at 40 MHz total
- and standard 2G/3G 900 MHz

> **n78 5G is not yet active on the site** as the remote radio units are missing from the monopole, as you can see below, and the 3500 MHz feeders

Reviewer: ```suggestion
**n78 5G is not yet active on the site** as the remote radio units are missing from the monopole, as you can see below, and the 3500 MHz feeders remain disconnected next to it. This RRU is set to be installed on 30 Jan 2022, according to information from [one.network](https://one.network/).
```

> remain disconnected next to it. This RRU is set to be installed on 30 Jan 2022, according to information from

Reviewer: ```suggestion
```

> [one.network](https://one.network/).

Reviewer: ```suggestion
```

![Missing RRU for 3500 MHz 5G between the passive antennas and 2300 MHz RRU](./missing-rru.png)

### Beam tilting

> Beam tilting is a way to **control the distance which a site can transmit to**. It works by changing the angle at which beams are transmitted at

Reviewer: ```suggestion
Beam tilting is a way to control the distance which a site can transmit at a given effective power level. It works by changing the angle at which beams are transmitted at from the antenna. In the past, this used to be done by physically moving the antenna elements inside the panel with motors, but this is now more commonly done through phase control.
```

> from the antenna. In the past, this used to be done by physically moving the antenna elements inside the panel with motors, but this is now more

Reviewer: ```suggestion
```

> commonly done thorugh **phase control**.

Reviewer: ```suggestion
```

> Phase control tilts beams by altering the time at which different antenna elements receive RF signals. This tilts the signal beam in a specific

Reviewer: ```suggestion
Phase control tilts beams by altering the time at which different antenna elements receive RF signals. This tilts the signal beam in a specific direction.
```

> direction.

Reviewer: ```suggestion
```

<FactBox>

> Beam tilting is particularly useful for controlling which sites are dominant in different areas, usually those with the most carrier frequencies

Reviewer: ```suggestion
Beam tilting is particularly useful for controlling which sites are dominant in different areas, usually those with the most carrier frequencies, or MIMO capabilities, or greatest backhaul capacity, in order to not overload lower capacity sites and to reduce interference from neighbouring cells.
```

> or MIMO capabilities or greatest backhaul capacity, in order to not overload smaller sites and to reduce interference from neighbouring cells.

Reviewer: ```suggestion
```

This beam tilting optimisation is often referred to as **coverage-capacity optimisation**.

</FactBox>

> On this Nokia Orion site, beam tilting is performed through the use of

Reviewer: ```suggestion
On this Nokia Orion site, beam tilting is performed through the use of [Kathrein 78211593 RETs](https://marketplace.shields-e.com/Product/Details/78211593), seen in between the feeders in the image below.
```

> [Kathrein 78211593 RETs](https://marketplace.shields-e.com/Product/Details/78211593), seen in between the feeders in the image below.

Reviewer: ```suggestion
```

![Image of Kathrein RET obscured by feeders into the passive antenna panel](./kathrein-rets.jpg)

# Attribution

Huge thank yous go to Peter Clarke, Derico, Allie, Jake (Undersc0re) and many others for help in piecing together sections of this article and
proofreading it.

> A big thanks also goes to the team at [ICS GROUP](https://www.linkedin.com/company/intelligent-communications-solutions-limited/) who installed

Reviewer: ```suggestion
A big thanks also goes to the team at [ICS GROUP](https://www.linkedin.com/company/intelligent-communications-solutions-limited/) who installed the new Orion site in Burgess Hill for letting me take photos and videos while they were working throughout the day.
```

> the new Orion site in Burgess Hill for letting me take photos and videos while they were working throughout the day.

Reviewer: ```suggestion
```
